docs/community/roadmap.md

```diff
@@ -128,12 +128,37 @@ Investigate to cooperate with argo to support job dependencies.
 * Owner: @yanglilangqun @Tammy-kunyu
 
 ## v1.6 (To be updated)
+### Support Hierarchy Queue(Part-2)
```

Comment:
We can not plan the 1.6/1.7 version at the version of 1.4. We only plan v1.5 at this time.

Reply:
> We can not plan the 1.6/1.7 version at the version of 1.4. We only plan v1.5 at this time.

I agree with not make future feature plans too far away. Just separate the work of three big feature to following three release.